WarOfEmpires.Domain/Players/Player.cs

```diff
@@ -379,6 +371,23 @@ public class Player : AggregateRoot {
             return buildingTotals;
         }
 
+        public int GetStaminaToHeal(int staminaToHeal) {
```

Comment:
Why does this have an argument staminaToHeal? Wouldn't you be able to determine this completely with properties of the player class?

Reply:
Argument removed, and calculation is now done in GetStaminaToHeal